openpdf/src/main/java/com/lowagie/text/Document.java

```diff
@@ -625,6 +626,47 @@ public boolean addCreationDate() {
             throw new ExceptionConverter(de);
         }
     }
+
+    /**
+     * Adds the current date and time to a Document.
+     *
+     * @return  <CODE>true</CODE> if successful, <CODE>false</CODE> otherwise
+     */
+    public boolean addCreationDate(PdfDate date) {
+        try {
+            return add(new Meta(Element.CREATIONDATE, date.toString()));
+        } 
+        catch (DocumentException de) {
+            throw new ExceptionConverter(de);
+        }
+    }
+    
+    /**
+     * Adds the current date and time to a Document.
+     *
+     * @return  <CODE>true</CODE> if successful, <CODE>false</CODE> otherwise
+     */
+    public boolean addModificationDate() {
+        try {
+            return add(new Meta(Element.MODIFICATIONDATE, new PdfDate().toString()));
+        } catch (DocumentException de) {
+            throw new ExceptionConverter(de);
+        }
+    }
```

Comment:
This is not a must have method, can be replaced by addModificationDate(PdfDate) and passing in current date.

API should be as simpler/cleaner as possible